### ISSUE TITLE
Computer Craft Mining Turtle has level 3 (because diamond pick)

### ISSUE THREAD

**Issue by spdkils** [NillerMedDild/Enigmatica2Expert#859]
**Modpack version**
Enigmatica 2 Expert

**Can the issue be reproduced?**
Yes

**Issue**
Tool progression ignores all fake players, so any mod that uses a fake player can dig ANY block no matter what your setting.
Also things like turtles use the actual diamond pick to calculate breaking of blocks. So the normal turtle uses a diamond pick (level 3) so does the advanced turtle(level3)
So the config you have makes it so if you 'HOLD' a turtle and break blocks you can break blocks up to 8 with holding an advanced turtle.

I think the only way to make it work how you would expect would be to allow the turtle to equip any tool. (Not an option that I'm aware of.)

Or change the recipe for the mining turtle to something else, and then set the diamond pick to the higher mining level,

Basically turtles are weaker than you set them to, and other block miners are basically god miners. Then again, by the time you can get either of them, it's almost a moot point.

**Comment by NillerMedDild** [comment]
The addition of the Mining Turtle to the tool config was just a last effort to make it work tbh :P I've discussed the issue with the mod dev, we never really found a good solution iirc.

I have done extensive work to make sure block breakers that can be configured are configured correctly, or disabled. I didn't know about fake players being considered god-level pickaxes :P but I don't think it's a large enough issue to disable them.

Is this issue just to inform me, or are you hoping to get something changed? :)

**Comment by spdkils** [comment]
I was hoping to have turtles a viable mining solution as opposed to the other completely automated ones. I just like turtles.
Only thing I can think of to fix this is to remove the diamond pick from game, sub out all diamond picks with a new item id, and then change the diamond pick to have a mining level 8 or whatever. All that effort  just to make the turtles work as expected. 👎 

I think continuum from FTB did a similar thing to solve a similar issue. Replaced all diamond picks with a stand in item that isn't the actual tool, then disabled the actual tool from crafting, and then anything that simulates a diamond pick (turtles is the only thing I can think of.) would be able to at least function in the over-world.

I could also just change the diamond pick to level 6, so it works overworld... and not mess with all this stuff. The fact the pick has durability 1 is probably sufficient to deal with most issues.

Anyone who wants to waste 3 diamonds to pick a single block to 'game' the system...

I'll probably just change the diamond pick at this point in my config... I think it's probably the easiest compromise in abilities, and still not breaking progression.

I did talk to the tool progressions mod dev about computer craft for a good 20 minutes today. No good options other than that. (Unless CC: Tweaked starts allowing equip of any tool, that would solve all these problems with turtles at least.)
